spec/facility_spec.rb

```diff
@@ -23,4 +29,173 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register' do
```

Comment:
I like that you used the describe block here to organize your code. All the it blocks nested inside this describe are related to the side effects the register_vehicle method have. However, I don't see the it blocks for the collected fees par. Make sure to include ALL the side effects in this section